api/src/controllers/users.js

```diff
@@ -143,7 +145,7 @@ function UsersControllerFactory (sequelize, auth, User, Invite, log, ledger, soc
 
       let dbUser
       let invite
-      yield sequelize.transaction(sequelize.Promise.coroutine(function * (t) {
+      yield sequelize.transaction(co.wrap(function * (t) {
```

Comment:
well in this case I use co only once to return a promise and I use normal `yield generator` inside the generator function instead of using `Promise.coroutine` and turning all of the generators to promises inside the function.